new-client/src/components/Window.js

```diff
@@ -339,13 +336,22 @@ class Window extends React.PureComponent {
   };
 
   maximize = () => {
-    const { onMaximize, onResize, allowMaximizedWindow } = this.props;
+    const {
+      globalObserver,
+      onMaximize,
+      onResize,
+      allowMaximizedWindow,
+      title,
+    } = this.props;
 
     getIsMobile() && this.rnd.updatePosition({ y: 0 });
 
     switch (this.state.mode) {
       case "minimized":
         // Enlarge back to "window" mode
```

Comment:
This comment isn't correct anymore?

Reply:
Can't find the comment. Wrong lines of code highlighted?

Reply:
Woops, i think it was "// Enlarge back to "window" mode" i was on about

Reply:
Unfortunately, I don't quite follow you. What's wrong with the current comment?

Reply:
I don't really remember what i meant hehe. Just leave it as is! :)

Reply:
Great! :)